lib/views/teach_word_view.dart

```diff
@@ -0,0 +1,356 @@
+                                        textAlign: TextAlign.center,
+                                        style: TextStyle(
+                                          fontSize: 17.5,
+                                          color: Color.fromRGBO(
+                                              245, 245, 220, 100),
+                                          fontFamily: 'Serif',
+                                        )),
+                                    const Text('寫字',
+                                        textAlign: TextAlign.center,
+                                        style: TextStyle(
+                                          fontSize: 17.5,
+                                          color: Color.fromRGBO(
+                                              245, 245, 220, 100),
+                                          fontFamily: 'Serif',
+                                        )),
+                                    const Text('邊框',
+                                        textAlign: TextAlign.center,
+                                        style: TextStyle(
+                                          fontSize: 17.5,
+                                          color: Color.fromRGBO(
+                                              245, 245, 220, 100),
+                                          fontFamily: 'Serif',
+                                        )),
+                                    const Text('重新',
+                                        textAlign: TextAlign.center,
+                                        style: TextStyle(
+                                          fontSize: 17.5,
+                                          color: Color.fromRGBO(
+                                              245, 245, 220, 100),
+                                          fontFamily: 'Serif',
+                                        )),
+                                  ],
+                                ),
+                              ),
+                            ],
+                          ),
+                        ),
+                      );
+                    }),
+                  ),
+                ),
+                TeachWordTabBarView(
+                    sectionName: '用一用',
+                    content: Column(
+                      children: [
+                        Container(
+                          height: 300,
+                          alignment: Alignment.center,
+                          child: const Column(
+                            children: [
+                              Text('手機',
+                                  textAlign: TextAlign.center,
+                                  style: TextStyle(
+                                    fontSize: 74,
+                                    color: Color.fromRGBO(245, 245, 220, 100),
+                                    fontFamily: 'Serif',
+                                  )),
+                              Image(
```

Comment:
Fix the image overflow issue.
![image](https://github.com/dlion168/learning-to-read-chinese/assets/22394272/1a996c9d-e79b-433d-be57-d72e42764503)